donut/modules/courses/routes.py

```diff
@@ -178,3 +178,24 @@ def scheduler_mine(year, term):
 
     return flask.jsonify(
         helpers.get_user_scheduler_sections(username, year, term))
+
+
+@blueprint.route('/1/scheduler/edit_notes', methods=['POST'])
+def edit_notes():
+    username = flask.session.get('username')
+    if not username:
+        return flask.jsonify({
+            'success': False,
+            'message': 'Must be logged in to save'
+        })
+    helpers.edit_notes(username,
+                       flask.request.form.get('courseId'),
+                       flask.request.form.get('notes'))
```

Comment:
Is there a way to delete notes? Maybe they should be deleted if set to the empty string?